python/GafferTest/ComputeNodeTest.py

```diff
@@ -368,5 +368,66 @@ def testInternalConnections( self ) :
 		self.assertEqual( n["out"].getValue(), a["sum"].getValue() )
 		self.assertEqual( n["out"].hash(), a["sum"].hash() )
 
+	def testErrorSignal( self ) :
+
+		b = GafferTest.BadNode()
+		a = GafferTest.AddNode()
+		a["op1"].setInput( b["out3"] )
+
+		cs = GafferTest.CapturingSlot( b.errorSignal() )
+
+		self.assertRaises( RuntimeError, b["out1"].getValue )
+		self.assertEqual( len( cs ), 1 )
+		self.assertTrue( cs[0][0].isSame( b["out1"] ) )
+		self.assertTrue( cs[0][1].isSame( b["out1"] ) )
+		self.assertTrue( isinstance( cs[0][2], str ) )
+
+		self.assertRaises( RuntimeError, a["sum"].getValue )
+		self.assertEqual( len( cs ), 2 )
+		self.assertTrue( cs[1][0].isSame( b["out3"] ) )
```

Comment:
I'm probably reading it wrong or misunderstanding, but I thought this would be `a["sum"]`